cordapp-cpk2/src/main/java/net/corda/plugins/cpk2/DependencyCalculator.java

```diff
@@ -236,6 +261,59 @@ public void calculate() {
         _providedJars.disallowChanges();
     }
 
+    @NotNull
+    private static Set<File> getCpbs(ResolvedConfiguration externalConfiguration) {
+        Set<File> cpbs = new HashSet<>();
+        for (File file: externalConfiguration.getFiles()) {
+            if (!file.getAbsolutePath().contains("/.")) {
+                File dir = file.getAbsoluteFile().getParentFile();
+                for (File siblingFile: Objects.requireNonNull(dir.listFiles())) {
+                    if (siblingFile.isFile() && siblingFile.getAbsolutePath().endsWith(".cpb")) {
+                        cpbs.add(siblingFile);
+                    }
+                }
+            }
+        }
+        return cpbs;
+    }
+
+    @NotNull
+    private static Set<File> extractJarsFromCpb(Set<File> cpbs, Set<String> cordappFileNames) throws IOException {
+        Path pathToDir = Files.createTempDirectory("jars");
+        pathToDir.toFile().deleteOnExit();
```

Comment:
I thought using the built-in `deleteOnExit` made more sense than making our own implementation since the files will be used in a separate class from where we create them and passing the data between them would make things more complicated.